### ISSUE TITLE
Coq commands don't work in Goal and Info buffers

### ISSUE THREAD

**Issue by j-hui** [whonore/Coqtail#101]
Commands like `:Coq Check` and `:Coq Locate` etc. (and their corresponding keybinds) are local only to the buffer where the Coq script is, but it's often useful to have these commands to explore the proof context and output buffers.

Right now, the workaround I have is to yank whatever term I need into a register, switch my window focus back over to the Coq script, and either run `:Coq Check <c-r>"` or paste it into the Coq script and manually invoke the vernacular.

**Comment by whonore** [comment]
I agree that this would be nice to have. What's blocking it is that currently a lot of the code has the assumption that it's called from the main buffer baked in. That was mostly just a simplifying assumption to get things working initially but it shouldn't be too hard to remove now.
